main/getting-started/before-using-agoric.md

```diff
@@ -15,42 +16,43 @@ git clone https://github.com/Agoric/agoric-sdk
 cd agoric-sdk
 yarn install
 yarn build
-yarn link-cli ~/bin/agoric
-agoric --version
+yarn link-cli ~/bin/agoric # creates an Agoric directory to reference
+export PATH=$PATH:~/bin # adds the parent directory to your PATH
+agoric --version # should print the version number of the SDK
 ```
 
-Then proceed to [starting a project](/getting-started/start-a-project.md).
+Now you are ready proceed to [starting a project](/getting-started/start-a-project.md).
 
-A more detailed explanation follows.
+If `agoric` is not found, then you may need to add the parent directory of where your `yarn link-cli` command created a folder. For example, if you created a folder called `Agoric` in your `/bin` directory, you may need to add `/bin` to your PATH.
 
 ::: tip Watch: Prepare Your Agoric Environment (November 2020)
 This presentation is a good overview of the Agoric SDK setup process,
 though a few details are out of date:
 
- - node version: 12.x is too old; use 14.15.0 or higher
- - skip `git checkout hackathon-2020-11`; use the default `master` branch
+- node version: 12.x is too old; use 14.15.0 or higher
+- skip `git checkout hackathon-2020-11`; use the default `master` branch
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/w0By22jYhJA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+**Note:** Video omits adding the Agoric SDK to your PATH.
 :::
 
 ## Platform: Linux shell or equivalent
 
 The Agoric SDK is supported on
 <a href="https://en.wikipedia.org/wiki/Linux">Linux</a>,
-<a href="https://www.apple.com/macos/">MacOS</a>, or
-<a href="https://docs.microsoft.com/en-us/windows/wsl/">Windows Subsystem for Linux (WSL).</a>
-
- - To open a terminal on Macs, see **Applications>Utilities>terminal** in the **Finder**.
- - To launch a bash shell at a specific folder on Windows 10:
-   1. Navigate to that folder in File Explorer.
-   2. Click the address bar while in that folder.
-   3. Type <code>bash</code> in the address bar and press <b>Enter</b>
+<a href="https://www.apple.com/macos/">MacOS</a>,
+<a href="https://docs.microsoft.com/en-us/windows/wsl/">Windows Subsystem for Linux (WSL)</a>, and
+<a href="https://www.docker.com/">Docker</a>.
 
+- To open a terminal on MacOS, see **Applications>Utilities>terminal** in the **Finder**.
+- To launch a terminal on Linux, use the **Terminal** application.
+- To access WSL from Windows, visit the [WSL documentation](https://docs.microsoft.com/en-us/windows/wsl/).
+- To use a Docker container, see the [Docker documentation](https://docs.docker.com/) and the [Agoric container images](https://hub.docker.com/u/agoric).
```

Comment:
Based on discussion with @michaelfig , I'm not sure we want to support customers in using those docker images for development. We sort of supported them during our beta release for clients / wallets for folks that didn't want to install the SDK. But these docs are not for that audience.